app/src/main/java/com/github/multimatum_team/multimatum/viewmodel/UserViewModel.kt

```diff
@@ -37,7 +37,7 @@ class UserViewModel @Inject constructor(
         _user
 
     fun signOut() = viewModelScope.launch {
-        Log.d("UserViewModel", "signing out from ${_user.value}")
+        LogUtil.debugLog("signing out from ${_user.value}")
```

Comment:
I see that you are using the methods from `LogUtil`, are they working correctly now?

Reply:
Yes, thank you for the fix